app/Alspotify.js

```diff
@@ -111,6 +111,19 @@ class Alspotify {
 			const lyricData = await alsong(this.info.artist, this.info.title, { playtime: Number(this.info.duration) });
 			const lyrics = await alsong.getLyricById(lyricData[0].lyricId);
 
+			if(lyrics.length === 0) {
```

Comment:
I think we should check the length of the `lyricData`, as the `lyrics` is not an array and `lyricData[0]` could be undefined